src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -458,4 +460,16 @@ public static String convertToRequiredFormat(String date, String hour, String mi
 
     }
 
+    public static TimeZone getTimeZoneFromDoubleOffset(double sessionTimeZone) {
+        String sign = sessionTimeZone > 0 ? "+" : "";
+        int hours = (int) sessionTimeZone;
+        int minutes = (int) ((Math.abs(sessionTimeZone) - Math.floor(Math.abs(sessionTimeZone))) * 60);
+        String offset = Integer.toString(hours);
+        if (minutes != 0) {
+            offset = String.format("%s:%s", Integer.toString(hours), Integer.toString(minutes));
```

Comment:
There are better ways to format this string, see https://docs.oracle.com/javase/7/docs/api/java/util/Formatter.html#syntax.
One way to simplify the code is to use `String.format("GMT%+03d:%02d", hours, minutes)`